new/LIT6718PPAqweyasat.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6718PPAqweyasat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against <foreign xml:lang="gez">ʾaqʷəyāṣāt</foreign> (shinbone?) and magicians</title>
```

Comment:
Unfortunately  I cannot figure out the implications of ጸሎተ፡ አቍያጻት and why shinbones are linked to magicians